keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ArtifactRepositoryTests.kt

```diff
@@ -786,12 +786,12 @@ abstract class ArtifactRepositoryTests<T : ArtifactRepository> : JUnit5Minutests
       }
 
       test("retrieves successfully") {
-        val artifactVersion = subject.getArtifactInstance(artifact1.name, artifact1.type, version1, SNAPSHOT)!!
```

Comment:
I don't think renaming this adds a ton of value, and it makes the PR longer / a bit harder to review. You don't have to revert this, but please keep this in mind going forward. What I typically do is change names if I'm trying to figure something out, and then when I figure it out change them back unless I think it's very miss-named. This is more of a style thing, but I'm commenting because I've seen you do this in several recent PRs.

Reply:
Yeah, I know... I kinda regretted it after making the change. I just have a hard time looking past inconsistent naming like this. But I could totally have made a separate PR. Happy to undo these if you like.

Reply:
Reverted. I'll open a separate one to rename

Reply:
I'd just like you to weigh the cost of renaming the variables in every file which you think they are a little bit wrong. It's a tax on the people working on the codebase in general. Sometimes it's worth it, I agree! But I think sometimes it's smoother (when working on a shared codebase) to live with a little bit of idiosyncrasy. Up to you here on what you'd like to do.

Reply:
I hear you. I try to keep that in mind, but I recognize I'm a bit obsessed with detail. Will try and tune that down going forward (or at least ask first if it's worth making the change).